src/features/DebugSession.ts

```diff
@@ -87,7 +87,7 @@ export class DebugSessionFeature implements IFeature, DebugConfigurationProvider
                 }
             }
 
-            if (!config.runspaceId) {
+            if (!config.runspaceId && !config.runspaceName) {
                 config.runspaceId = await vscode.commands.executeCommand("PowerShell.PickRunspace", config.processId);
```

Comment:
Should we change the runspace picker to allow picking by name instead of id?  If not for v1.x, then perhaps for 2.x where  we only support >= 5?

Reply:
![image](https://user-images.githubusercontent.com/2644648/57314776-24289700-70a7-11e9-8f42-5f94a795cfb4.png)

It says the name in the runspace picker so I think we're good here